global.yaml

```diff
@@ -21,6 +21,9 @@ global:
   istioNamespace: istio-control
   configNamespace: istio-control
 
+  # Applications namespace list pilot manages, separated by comma, if it contains multiple namespaces.
+  appNamespace: ""
```

Comment:
shouldn't this be a pilot specific config in the pilot specific values.yaml file?

Reply:
+1, and also please no comma separate list in yaml !